geoportailv3/static/js/draw/drawnfeaturesservice.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @fileoverview Provides a selected features service useful to share
+ * information about the selected feature throughout the application.
+ */
+
+goog.provide('app.DrawnFeatures');
+
+goog.require('app');
+goog.require('ol.Collection');
+
+
+/**
+ * @typedef {ol.Collection<ol.Feature>}
+ */
+app.DrawnFeatures;
+
+
+/**
+ * The selected features collection.
+ * @private
```

Comment:
Very minor: you can add `@type {app.DrawFeatures}`, and `@const` to tell the compiler that this won't change.